HW31-flask/flask_app/views.py

```diff
@@ -41,63 +47,69 @@ def books():
                  "Приключения Шерлока Холмса", "Дубровский", "Горе от ума", "Драма на охоте",
                  "Капитанская дочка", "Униженные и оскорблённые", "Воскресение", "Дворянское гнездо",
                  "Рассказы", "Триумфальная арка", "Подросток", "Старик и море"]
-    if request.args:
-        lst3 = sample(lst_books, k=int(request.args.get('count')))
+    if session.get('user'):
+        if request.args:
+            lst3 = sample(lst_books, k=int(request.args.get('count')))
```

Comment:
You have here an internal server error if the user passes some arguments but not `count`.